api/src/pcapi/core/offerers/models.py

```diff
@@ -276,6 +276,15 @@ def demarchesSimplifieesApplicationId(self) -> int | None:
 
         return self.bankInformation.applicationId
 
+    @property
+    def hasPendingBankInformationApplication(self) -> bool:
+        return (
+            bool(self.bankInformation)
+            and self.bankInformation.status != BankInformationStatus.REJECTED
+            and not self.bic
+            and not self.iban
```

Comment:
(question) Y a-t-il des cas où le `BankInformation` est à l'état "accepté" **sans** IBAN ni BIC ? Ça me semble étrange. Naïvement, j'aurais simplement fait `return bool(self.bankInformation) and self.bankInformation.status == DRAFT`.

Reply:
J'ai fait de l'iso-fonctionnel, mais on pourrait simplifier.